src/daemon/config.rs

```diff
@@ -486,7 +488,7 @@ impl Config {
         let auth_users = None;
         #[cfg(feature = "multi-user")]
         let auth_openidconnect = None;
-        let ca_refresh = 1;
+        let ca_refresh = if enable_ca_refresh { 1 } else { 86400 };
```

Comment:
Should `ca_refresh` be renamed `ca_refresh_seconds` or something here to express the unit of the value?